### PR TITLE
PlaceID refresh API

### PR DESCRIPTION
**Pull request by danielabreo**
`[GET] /data/update/placids`: This endpoint refreshes all placeID's in the `Organizations` table.

The Cloud Scheduler job is accessible here: https://pantheon.corp.google.com/cloudscheduler?project=gsi-step-capstone.
When deployed, logs detailing the result of every job will be available at this link.

### CODE REVIEW

File: routes/data.js [hunk 1]
```diff
@@ -248,3 +249,38 @@ exports.organizationsPost = async function (req, res) {
...
+  const organizations = await firestore.collection(resolveCollectionName('Organizations')).get();
+
+  let tasksCompleted = '';
+
+  await Promise.all(
+    organizations.docs.map(async (doc) => {
+      const oldPlaceID = doc.data().placeID;
+      const oldPlaceName = doc.data().name;
+
+      if (oldPlaceID != undefined && oldPlaceID != '') {
```
Comment (brucezheng) [inline routes/data.js:264]: You can replace

```x != undefined && x != ''```

with

```!!x```

File: routes/data.js [hunk 2]
```diff
@@ -248,3 +249,38 @@ exports.organizationsPost = async function (req, res) {
...
+        // If the current entry has a placeID, check if it is the latest one.
+        const placeJSON = await (
```
Comment (brucezheng) [inline routes/data.js:266]: Why do you need two nested ```await```s?

Comment (danielabreo) [reply routes/data.js:266]: Both the fetch() and res.json() functions need awaits: https://gist.github.com/msmfsd/fca50ab095b795eb39739e8c4357a808#gistcomment-2085059

File: routes/data.js [hunk 3]
```diff
@@ -274,3 +275,38 @@ exports.organizationsPost = async function (req, res) {
...
+exports.updatePlaceIDs = async function (req, res) {
+  // Get all entries from the Organizations table.
+  const organizations = await firestore.collection(resolveCollectionName('Organizations')).get();
+
+  let tasksCompleted = '';
+
+  await Promise.all(
+    organizations.docs.map(async (doc) => {
+      const oldPlaceID = doc.data().placeID;
+      const oldPlaceName = doc.data().name;
```
Comment (brucezheng) [inline routes/data.js:288]: If we aren't updating the place name why is this called ```oldPlaceName```?

**Pull request merged**